scripts/nlp_api_master.py

```diff
@@ -295,11 +513,76 @@
                         "keras_nlp.models.XLMRobertaClassifier.preprocessor",
                     ],
                 },
+                {
+                    "path": "xlm_roberta_masked_lm",
+                    "title": "XLMRobertaMaskedLM model",
+                    "generate": [
+                        "keras_nlp.models.XLMRobertaMaskedLM",
+                        "keras_nlp.models.XLMRobertaMaskedLM.from_preset",
+                        "keras_nlp.models.XLMRobertaMaskedLM.backbone",
+                        "keras_nlp.models.XLMRobertaMaskedLM.preprocessor",
+                    ],
+                },
+                {
+                    "path": "xlm_roberta_masked_lm_preprocessor",
+                    "title": "XLMRobertaMaskedLMPreprocessor layer",
+                    "generate": [
+                        "keras_nlp.models.XLMRobertaMaskedLMPreprocessor",
+                        "keras_nlp.models.XLMRobertaMaskedLMPreprocessor.from_preset",
+                        "keras_nlp.models.XLMRobertaMaskedLMPreprocessor.tokenizer",
+                    ],
+                },
             ],
         },
     ],
 }
 
+SAMPLERS_MASTER = {
+    "path": "samplers/",
+    "title": "Samplers",
+    "toc": True,
+    "children": [
+        {
+            "path": "samplers",
+            "title": "Sampler base class",
+            "generate": [
+                "keras_nlp.samplers.Sampler",
+                "keras_nlp.samplers.Sampler.get_next_token",
+            ],
+        },
+        {
+            "path": "beam_sampler",
+            "title": "Beam sampler",
+            "generate": ["keras_nlp.samplers.BeamSampler"],
+        },
+        {
+            "path": "contrastive_sampler",
+            "title": "Contrastive sampler",
+            "generate": ["keras_nlp.samplers.ContrastiveSampler"],
+        },
+        {
+            "path": "greedy_sampler",
+            "title": "Greedy sampler",
+            "generate": ["keras_nlp.samplers.GreedySampler"],
+        },
+        {
+            "path": "random_sampler",
+            "title": "Random sampler",
+            "generate": ["keras_nlp.samplers.RandomSampler"],
+        },
+        {
+            "path": "top_k_sampler",
+            "title": "Top-K sampler",
```

Comment:
Most titles above contain the actual symbol name. I think we want TopKSampler here (and elsewhere).